tests/tests_pytorch/core/test_datamodules.py

```diff
@@ -236,12 +242,12 @@ def test_full_loop(tmpdir):
     # validate
     result = trainer.validate(model, dm)
     assert dm.trainer is not None
-    assert result[0]["val_acc"] > 0.7
```

Comment:
Do you think we should revert these numbers but instead find another seed that satisfies the existing threshold?

Reply:
I think it's fine since these are not benchmarks...